jni/tests/test_util.h

```diff
@@ -108,17 +108,17 @@ namespace test_util {
     faiss::Index* FaissLoadFromSerializedIndex(std::vector<uint8_t>* indexSerial);
 
     faiss::IndexIDMap FaissAddData(faiss::Index* index,
-                                   std::vector<faiss::Index::idx_t> ids,
+                                   std::vector<faiss::idx_t> ids,
```

Comment:
All these are fixed in the main branch. Feel free to rebase your branch to get rid of these changes.

Reply:
++ LGTM, i'll sync upstream master branch and rebase it

Reply:
Synced upstream master branch into this branch. maybe [opensearch-project:feature/faiss-nsg-support](https://github.com/opensearch-project/k-NN/tree/feature/faiss-nsg-support) need sync to master branch for compare